qabelbox/src/androidTest/java/de/qabel/qabelbox/services/LocalQabelServiceTest.java

```diff
@@ -136,12 +136,7 @@ public void testSendAndReceiveDropMessage() throws QblDropPayloadSizeException,
         final CountDownLatch lock = new CountDownLatch(1);
 
         mService.sendDropMessage(dropMessage, recipientContact, senderIdentity,
-                new LocalQabelService.OnSendDropMessageResult() {
-                    @Override
-                    public void onSendDropResult(Map<DropURL, Boolean> deliveryStatus) {
-                        lock.countDown();
-                    }
-                });
+                deliveryStatus -> lock.countDown());
```

Comment:
This could actually be the case. Sometimes the build never finishes when the connectivity is bad.